dbt/models/proximity/proximity.dist_pin_to_new_construction.sql

```diff
@@ -0,0 +1,38 @@
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH new_construction AS (  -- noqa: ST03
```

Comment:
@ccao-jardine This is the query that constructs the set of new construction parcels we're comparing distance to. Do you want to try to include commercial and other classes in here as well?